ra-optimisation/search-tree-rao/src/test/java/com/farao_community/farao/search_tree_rao/RangeActionFilterTest.java

```diff
@@ -210,6 +222,37 @@ public void testFilterTsos() {
         assertTrue(filteredRangeActions.contains(pstde2));
     }
 
+    @Test
+    public void testFilterTsosWithAlignedPsts() {
```

Comment:
you should also create a test case where you can only have 2 tsos for example, and you chose fr and be because they have PSTs in the same group, even if there is a NL pst with a higher sensitivity than BE